.github/renovate/labels.json5

```diff
@@ -1,41 +1,37 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "packageRules": [
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
```

Comment:
I think we should retain the quotes on the renovate config due to `Property keys must be doublequoted [jsonc]`